woocommerce/wc-admin
# Fixed associated Order Number for refunds

Fixes #5867

This PR fixes the error related to not showing the associated order number when the order has a `refund` status.
This was happening in `Analytics` > `Orders` .


### Accessibility




### Screenshots
**Before**
![screenshot-constitutional-cow jurassic ninja-2021 02 23-10_36_49](https://user-images.githubusercontent.com/1314156/108876138-a9fb3680-75dc-11eb-8a19-a2770f40d563.png)

**Ater this PR**
![screenshot-constitutional-cow jurassic ninja-2021 02 23-10_37_36](https://user-images.githubusercontent.com/1314156/108876159-b1badb00-75dc-11eb-979e-09c5cdc662f9.png)



### Detailed test instructions:

0. In a store with refunded orders.
1. Check out the `main` branch.
2. Go to `Analytics` > `Orders`
3. Set the `Date Range` filter in order to cover the refunded order date.
4. Verify the associated Order Number for refunds is not visible.

5. Check out this branch (`git checkout fix/5867`).
6. Disable the report cache. You can do it by installing and activating [this plugin](https://gist.github.com/octaedro/693f3d64c201cc061fc936adf3c64429).
7. Verify that now the associated order number and the related products are visible.
8. Verify also that there aren't any JS warnings related to an `undefined` value for `totalResults`.
